src/AddItems.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { capitalize } from './lib/utility';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const [itemFreq, setItemFreq] = useState(7);
+  const [formValidationVisual, setFormValidationVisual] = useState(false);
+
+  const updateGroceryItem = (event) => {
+    setGroceryItem(capitalize(event.target.value));
+    setFormValidationVisual(true);
+  };
+
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: itemFreq,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
+    setGroceryItem('');
+    setItemFreq(7);
+    setFormValidationVisual(false);
+  };
+
+  const radioBtnHandler = (event) => {
+    setItemFreq(+event.target.value);
+    setFormValidationVisual(true);
+  };
+
+  return (
+    <div>
+      <form id="test-form" onSubmit={submitGroceryItem}>
+        <label htmlFor="item">Item name:</label>
+        <input
+          type="text"
+          id="item"
+          value={groceryItem}
+          required={formValidationVisual}
+          onChange={updateGroceryItem}
+        />
+        <input
```

Comment:
Take a look at the wireframes for this page. I think the radio buttons should be wrapped in a fieldset:
![image](https://user-images.githubusercontent.com/10378331/114867002-e287e700-9db9-11eb-87f2-3f610befb4db.png)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset